http_service/Dockerfile

```diff
@@ -1,14 +0,0 @@
-ARG BUGBUG_VERSION=latest
```

Comment:
I'd keep the Dockerfiles under http_service, so we can more easily move it to a separate repository if we want in the future.

Reply:
Done